src/hotspot/share/jfr/jfr.cpp

```diff
@@ -100,6 +100,10 @@ void Jfr::on_set_current_thread(JavaThread* jt, oop thread) {
   JfrThreadLocal::on_set_current_thread(jt, thread);
 }
 
+void Jfr::initialize_primordial_thread(JavaThread* jt) {
+  JfrThreadLocal::initialize_primordial_thread(jt);
```

Comment:
The main thread is generally not the process primordial thread. Can we use main_thread or starting_thread here to keep semi-consistent with the Thread/Threads code terminology? Thanks

Reply:
Ok.

Reply:
Updated.